README.md

```diff
@@ -1,16 +1,74 @@
 # Forest Rails Liana [![Build Status](https://travis-ci.org/ForestAdmin/forest-rails.svg?branch=master)](https://travis-ci.org/ForestAdmin/forest-rails)
-The official Rails liana for Forest.
+
+The official Rails liana for Forest.  
+Forest is a modern admin interface (see the [live demo](https://app.forestadmin.com/login?livedemo)) built for projects of all major web frameworks.  
+forest-rails gem is the liana (=connector) that makes Forest admin work on any rails app.
```

Comment:
`forest_liana` is the gem that makes Forest admin work on any Rails application.